nncf/common/utils/decorators.py

```diff
@@ -39,8 +39,7 @@ def wrapped_f(*args, **kwargs):
                     nncf_logger.warning(
                         f"{ex.msg} Please install NNCF package with dev "
                         "extra. Use one of the following commands "
-                        '"pip install .[dev]" running from the repository '
-                        'root directory or "pip install nncf[dev]"'
+                        '"pip install -r requirements-dev.txt" running from the repository.'
```

Comment:
I wonder how the user will do this if he installed nncf from pip. Could you save the behavior from develop for this case?

Reply:
return nncf[dev]